backend/routes/index.js

```diff
@@ -0,0 +1,85 @@
+var express = require('express');
+var router = express.Router();
+var { generateToken, sendToken } = require('../utils/token.utils');
+var passport = require('passport');
+var config = require('../config');
+var request = require('request');
+require('../passport')();
+
+router.route('/auth/twitter/reverse')
+    .post(function(req, res) {
+        request.post({
+            url: 'https://api.twitter.com/oauth/request_token',
+            oauth: {
+                oauth_callback: "http%3A%2F%2Flocalhost%3A3000%2Ftwitter-callback",
+                consumer_key: config.twitterAuth.consumerKey,
+                consumer_secret: config.twitterAuth.consumerSecret
+            }
+        }, function (err, r, body) {
```

Comment:
This looks like a callback function. If it is, you could probably write is using async/await or promises to make it look better